packages/mask/src/components/InjectedComponents/ProfileTabContent.tsx

```diff
@@ -63,11 +145,46 @@ export function ProfileTabContent(props: ProfileTabContentProps) {
         [SocialAddressType.NEXT_ID],
     )
 
+    const { value: currentVisitingSocialIdentity, loading: loadingCurrentVisitingSocialIdentity } =
+        useCurrentVisitingSocialIdentity()
+
+    const { value: personaProof, retry: retryProof } = useAsyncRetry(async () => {
+        if (!currentVisitingSocialIdentity?.publicKey) return
+        return NextIDProof.queryExistedBindingByPersona(currentVisitingSocialIdentity?.publicKey)
```

Comment:
Adding the `bindings` property to `SocialIdentity` and removing this redundant query is better.